sal_interfaces/Guider/Guider_Commands.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>Guider</Subsystem>
+    <EFDB_Topic>Guider_command_startGuiding</EFDB_Topic>
+    <Description>Start guiding using the input information. The command returns as soon as guiding starts, before the first interation is completed. If it fails to find a guide star in the provided guider region, the CSC will go to fault state.</Description>
+    <item>
+      <EFDB_Name>roiX</EFDB_Name>
+      <Description>Vector with 8 elements specifying the center of the guider ROI window in the x-axis for each guider sensor. If an element of the array is zero, that guider will be ignored.</Description>
+      <IDL_Type>long</IDL_Type>
```

Comment:
Center can be on fractional pixels. Assume you choose odd size for ROI (e.g. 3x3), your pixel center would be 1.5 in x and y (placed on origin). It's probably better to specify ROI using roiXleft,roiXright,roiYbottom,roiYupper coordinates.

Reply:
good point!